bin/espruino-cli.js

```diff
@@ -554,6 +554,11 @@ function startConnect() {
   if ((!args.file && !args.updateFirmware && !args.expr) || (args.file && args.watchFile)) {
     if (args.ports.length != 1)
       throw new Error("Can only have one port when using terminal mode");
+
+    if ( args.ports[0].name.substr(0, 6)==="tcp://" ) {
+      Espruino.Config.SERIAL_TCPIP=args.ports[0].name; // ensure node serial will return this uri
```

Comment:
This looks like a good way of handling it - but could you maybe add this into `setupConfig` and make it search the whole array of ports? That way it'd be possible to specify multiple TCP/IP ports if you wanted to deploy to multiple devices at once